data/src/input.rs

```diff
@@ -74,7 +74,7 @@ impl Input {
 }
 
 #[derive(Debug, Clone)]
-enum Content {
+pub enum Content {
     Text(String),
```

Comment:
I've locked this module back down 322ec2e and added an appropriate builder method. We don't need to expose these internals.